Tic_Tac_Toe_for_nerds.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+
+def unique_players(inputs):
+    l = list()
+    for row in inputs:
+        for i in row:
+            l.append(i)
+    s = set(l)
```

Comment:
you could potentially create a `set` initially instead of `list`, but not sure if that works